Dockerfile

```
@@ -13,7 +14,7 @@ RUN pip install --upgrade pip && pip install wheel
 RUN pip wheel . --wheel-dir=/wheels
 
 # Install from wheels
-FROM apeworx/ape:stable
+FROM ghcr.io/apeworx/ape:${APE_VERSION:-latest-slim}
```

Will that extra dash appear for the default?
```suggestion
FROM ghcr.io/apeworx/ape:${APE_VERSION:latest-slim}
```

---

yes it will, that's how you designate a default variable in bash and dockerfiles